RELEASES.md

```diff
@@ -6,8 +6,87 @@ Unreleased.
 
 ### Added
 
+* Allow the MPL-2.0 and OpenSSL licenses in dependencies of wasmtime.
+  [#6136](https://github.com/bytecodealliance/wasmtime/pull/6136)
+
+* The initial integration between winch and wasmtime has landed behind a cargo
+  feature.
+  [#6119](https://github.com/bytecodealliance/wasmtime/pull/6119)
```

Comment:
Oh this ended up landing after the branch point so this won't be here until 8.0.0